rules/private/phases/phase_zinc_compile.bzl

```diff
@@ -72,7 +73,7 @@ def phase_zinc_compile(ctx, g):
         ] + [zinc.deps_files for zinc in zincs],
     )
 
-    outputs = [g.classpaths.jar, mains_file, analysis_store, used, tmp]
+    outputs = [g.classpaths.jar, mains_file, analysis_store, analysis_store_text, used, tmp]
```

Comment:
Do we know how expensive this is to do?
Additionally, do we have a visible place to put a TODO to remove this logging once we've figured out the reproducibility problems?

Reply:
I don't think it's too expensive to do, but don't have numbers to back that up. I didn't notice any difference with it on or off, but I also wasn't taking measurements.

I've got a reminder set for first week of September to come disable this if we no longer need it.